### ISSUE TITLE
Issue with slurm.

### ISSUE THREAD

**Issue by CalvinDSeamons** [hpc/pavilion2#355]
I've noticed some odd behavior while running tests that use the slurm scheduler. 
If I have a test that should error the below events occur. 

slurm segment from my test:
```yaml
    slurm:
      num_nodes: 4
      qos: blahblah_uh_oh // this should cause a error?
```
If we run that test we get:
```bash
[calvin@thing build]$ pav run truchas-prebuilt
Resolving Test Configs: 100%
Creating Test Runs: 100%
BUILD_REUSED: 1
```
NOTE: BUILD is last message to appear, no series start, test started, etc, as it use to.
`pav status` will generate:
```bash
[calvin@thing build]$ pav status
 Test statuses
---------+-----------------------+-----------------+----------+----------------------------------------
 Test id | Name                  | State           | Time     | Note
---------+-----------------------+-----------------+----------+----------------------------------------
 27      | truchas-prebuilt.base | SCHED_CANCELLED | 16:01:58 | Job was never started.
```
and no kickoff.log will be generated. 

I don't believe this was intentional so something changed in a recent merge request (maybe...).

**Comment by kjeverson** [comment]
Francine's #360 fixes all of this. The `qos` raised an error, but the message wasn't displayed. But, that is why you got the `Job was never started.` message and since it wasn't started it won't have a `kickoff.log`.

**Comment by CalvinDSeamons** [comment]
Addressed by #360
